ui/components/multichain/connected-site-menu/connected-site-menu.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
```

Comment:
Using `useRef` here not sure why it's using `useState` in the component-library docs 🤔